server/settings/__init__.py

```diff
@@ -13,6 +13,7 @@ class Config:
     }
     TESTING = False
     CACHE_KEY_PREFIX = 'ok-cache'
+    SENDGRID_API_KEY = os.getenv('SENDGRID_API_KEY', '')
```

Comment:
It's technically unnecessary to have this in settings since the sender uses os.getenv directly.